src/VCDiff/Decoders/WindowDecoder.cs

```diff
@@ -55,11 +61,18 @@ internal class WindowDecoder
         /// </summary>
         /// <param name="dictionarySize">the dictionary size</param>
         /// <param name="buffer">the buffer containing the incoming data</param>
-        public WindowDecoder(long dictionarySize, IByteBuffer buffer)
+        /// <param name="maxWindowSize">The maximum target window size in bytes</param>
+        public WindowDecoder(long dictionarySize, IByteBuffer buffer, int maxWindowSize = DefaultMaxTargetFileSize)
```

Comment:
```suggestion
        public WindowDecoder(long dictionarySize, IByteBuffer buffer, int maxWindowSize = WindowDecoder.DefaultMaxTargetFileSize)
```

Reply:
Visual Studio suggests that I don't prefix this. Are you sure ?

Reply:
Yes, the convention I use is to fully qualify local static members.